### ISSUE TITLE
Version 1.0.30 throws Exception @ Lumen

### ISSUE THREAD

**Issue by padys** [Folkloreatelier/laravel-graphql#223]
After upgrading "folklore/graphql" from 1.0.29 to 1.0.30 in Lumen (version 5.4.7) appears Exception:

```
Call to a member function parameter() on array
in Request.php (line 452)
at Request->route('graphql_schema')in GraphQLController.php (line 10)
at GraphQLController->__construct(object(Request))
at ReflectionClass->newInstanceArgs(array(object(Request)))in Container.php (line 762)
at Container->build('\\Folklore\\GraphQL\\GraphQLController')in Container.php (line 608)
at Container->resolve('\\Folklore\\GraphQL\\GraphQLController')in Container.php (line 575)
at Container->make('\\Folklore\\GraphQL\\GraphQLController')in Application.php (line 210)
at Application->make('\\Folklore\\GraphQL\\GraphQLController')in RoutesRequests.php (line 676)
at Application->callControllerAction(array(true, array('as' => 'graphql.graphiql', 'middleware' => array(), 'uses' => '\\Folklore\\GraphQL\\GraphQLController@graphiql'), array()))in RoutesRequests.php (line 643)
at Application->callActionOnArrayBasedRoute(array(true, array('as' => 'graphql.graphiql', 'middleware' => array(), 'uses' => '\\Folklore\\GraphQL\\GraphQLController@graphiql'), array()))in RoutesRequests.php (line 623)
at Application->Laravel\Lumen\Concerns\{closure}()in RoutesRequests.php (line 781)
at Application->sendThroughPipeline(array(), object(Closure))in RoutesRequests.php (line 624)
at Application->handleFoundRoute(array(true, array('as' => 'graphql.graphiql', 'middleware' => array(), 'uses' => '\\Folklore\\GraphQL\\GraphQLController@graphiql'), array()))in RoutesRequests.php (line 528)
at Application->Laravel\Lumen\Concerns\{closure}(object(Request))
at call_user_func(object(Closure), object(Request))in Pipeline.php (line 52)
at Pipeline->Laravel\Lumen\Routing\{closure}(object(Request))in StatsdRequestLogger.php (line 52)
at StatsdRequestLogger->handle(object(Request), object(Closure))in Pipeline.php (line 148)
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request))in Pipeline.php (line 32)
at Pipeline->Laravel\Lumen\Routing\{closure}(object(Request))in HTTPCache.php (line 18)
at HTTPCache->handle(object(Request), object(Closure))in Pipeline.php (line 148)
...
```

**Comment by dmongeau** [comment]
Should be fixed in 1.0.31

**Comment by padys** [comment]
Fix works. Thanks!
